L1Trigger/TrackFindingTracklet/src/FitTrack.cc

```diff
@@ -934,10 +934,12 @@ void FitTrack::execute() {
     bool match = false;
 
     if (indexArray[0] < matches1.size()) {
```

Comment:
once indexArray[0] < matches1.size() is part of `while` then this is also not needed